spec/factories/repayments.rb

```diff
@@ -5,7 +5,7 @@
     amount_paid { amount_due }
     interest_since_last_payment { rand(1..amount_due) }
     date_due { Faker::Date.between(Date.civil(2004, 01, 01), Date.today) }
-    loan_id { create(:loan).id }
+    project_id { create(:loan).id }
```

Comment:
See above.

Were the specs not passing when you started the branch? If so then it's someone else's job to fix them. I'll check with Fuzzy.